src/mono/mono/eglib/glib.h

```diff
@@ -1314,8 +1314,7 @@ gint
 g_clock_nanosleep (clockid_t clockid, gint flags, const struct timespec *request, struct timespec *remain);
 #endif
 
-//#define ENABLE_CHECKED_CASTS
-#ifdef ENABLE_CHECKED_CASTS
+#ifdef ENABLE_CHECKED_BUILD_CASTS
```

Comment:
What exactly is this checked option doing ? The defines below seem to not do anything in case of overflow.